engine/table/src/main/java/io/deephaven/engine/table/impl/sources/regioned/ColumnRegion.java

```diff
@@ -18,6 +18,13 @@ default long firstRowOffset() {
         return 0;
     }
 
+    //TODO: EXPERIMENT:
+
+    /**
+     * Poison the region -- any further reads will fail.
+     */
+    void poison();
```

Comment:
"invalidate" - this is user-facing, better naming
This implementation will work, but it's kind of aggressive. We only need to prevent new reads; reads from cache are totally cool.

Reply:
I think that we agreed to stay aggressive between @rcaudy @cpwright and @abaranec